src/__test__/redux/selectors/getNumberOfCellsInGrouping.test.js

```diff
@@ -1,4 +1,4 @@
-import getNumberOfCellsInGrouping from '../../../redux/getters/getNumberOfCellsInGrouping';
+import getNumberOfCellsInGrouping from '../../../redux/selectors/getNumberOfCellsInGrouping';
```

Comment:
This is going to be refactored into the corresponding selector shape (using local selectors, etc) in a different aprt of this ticket